### DISCUSSION TITLE
xarray not opening any dataset

### DISCUSSION THREAD

**Discussion by mmoranv** [giswqs/geog-312#119] [Q&A] [answered]
Hello! I completed lab 6 on VS Code, but the same code is unable to open the sea_temperature file on Exercise 1: 
![image](https://github.com/user-attachments/assets/fed909e1-b11d-46db-b455-a600f2bdee86)

The file is indeed in the location I indicate, but the code doesn't read the file. The code also doesn't work with the URL. Must be something dumb, but maybe you can see it more clearly. Thanks!

**Comment by giswqs** [comment, marked as answer]
Please watch this lecture: https://youtu.be/xIWk9b4Cjug
I covered this issue at the beginning of the lecture.

**Reply by mmoranv** [reply]
> Please watch this lecture: https://youtu.be/xIWk9b4Cjug I covered this issue at the beginning of the lecture.

Thank you Professor! Sorry it took me so long to look at this. It's not working for me. I tried using leafmap to download and I think it gets stuck? In any case, xarray is unable to open it, even after I install the netcdf4 dependency recommended by Gemini :/

![image](https://github.com/user-attachments/assets/de69abc8-09cd-4753-b052-9c44ef91541b)

**Reply by mmoranv** [reply]
@giswqs Hello Professor, I still have the error. Any workarounds?
